src/styles/global-font-style.scss

```diff
@@ -80,9 +80,9 @@
 }
 
 .font_hero_background {
-  color: rgba(254, 254, 254, 0.07);
```

Comment:
The 4th arg in the rgba means alpha channel, it should be a decimal around 0 to 1. You can check the UAT [taotify-website.vercel.app](https://taotify-website.vercel.app/), the background font is now missing, and I think you should turn it back.

Reply:
And also the background color. Please check one more time thx.